src/nodejs/lambda-handlers/submission.js

```diff
@@ -118,7 +118,10 @@ async function metadataMethod(event, user) {
 async function saveMethod(event) {
   const { form_id: formId, daac_id: daacId, data } = event;
   const { id } = event;
+  const product = data.data_producer_info_name;
```

Comment:
produce and producer values are flipped.  I fixed this in my pr that is based on this branch.